src/Files.App/Dialogs/CreateShortcutDialog.xaml

```diff
@@ -0,0 +1,59 @@
+﻿<ContentDialog
```

Comment:
Can you create a new folder and put the xaml and view model in the folder? This will help organize the code and make it easier to work on it.

Reply:
Under `src/Files.App/Dialogs` is fine?

Reply:
Disregard this for now